2024-09-23 20:01:30,446 | WARNING | Неверный тип данных для объекта Runner
Traceback (most recent call last):
  File "C:\Users\bfi18\PycharmProjects\start_project\homeworks\Module_12\homework12_4.py", line 21, in test_run
    hum = runner_tour_exc.Runner(1, 12)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\bfi18\PycharmProjects\start_project\homeworks\Module_12\runner_tour_exc.py", line 6, in __init__
    raise TypeError(f'Имя может быть только строкой, передано {type(name).__name__}')
TypeError: Имя может быть только строкой, передано int
2024-09-23 20:01:30,447 | WARNING | Неверная скорость для Runner
Traceback (most recent call last):
  File "C:\Users\bfi18\PycharmProjects\start_project\homeworks\Module_12\homework12_4.py", line 11, in test_walk
    hum = runner_tour_exc.Runner('ksu', -12)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\bfi18\PycharmProjects\start_project\homeworks\Module_12\runner_tour_exc.py", line 11, in __init__
    raise ValueError(f'Скорость не может быть отрицательной, сейчас {speed}')
ValueError: Скорость не может быть отрицательной, сейчас -12
